app/src/main/java/ch/epfl/sdp/blindly/user/User.kt

```diff
@@ -298,17 +298,22 @@ class User private constructor(
             return null
         }
 
-        @RequiresApi(Build.VERSION_CODES.O)
         fun getAgeFromBirthday(birthday: String): Int {
-            val (day, month, year) = birthday.split('.')
-            return Period.between(
-                LocalDate.of(
-                    year.toInt(),
-                    month.toInt(),
-                    day.toInt()
-                ),
-                LocalDate.now()
-            ).years
+            val (day, month, year) = birthday.split('.').map { s -> s.toInt() }
+            val calendar = GregorianCalendar()
```

Comment:
This snippet is in two different places, maybe a date utils class could be helpful?

Reply:
I don't know if a class for one method is really necessary, but maybe a public method could be a good idea indeed !